en/mkdocs.yml

```diff
@@ -702,7 +702,7 @@ nav:
       - 'Securing Passwords in Configuration Files':
         - 'Encrypting Passwords with Cipher Tool': administer/encrypting-passwords-with-cipher-tool.md
         - 'Resolving Encrypted Passwords': administer/resolving-encrypted-passwords.md
-        - 'Carbon Secure Vault Implementation': administer/carbon-secure-vault-implementation.md
+        - 'Customizing Secure Vault': administer/customizing-secure-vault.md
```

Comment:
Please check if there are links from other files to this and replace all available instances with the new name.